backend/oasst_backend/models/message_embedding.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+from typing import List, Optional
+from uuid import UUID
+
+import sqlalchemy as sa
+import sqlalchemy.dialects.postgresql as pg
+from sqlmodel import ARRAY, Field, Float, SQLModel
+
+
+class MessageEmbedding(SQLModel, table=True):
+    __tablename__ = "message_embedding"
+    __table_args__ = (sa.PrimaryKeyConstraint("message_id", "model"),)
+
+    message_id: UUID = Field(sa_column=sa.Column(pg.UUID(as_uuid=True), sa.ForeignKey("message.id"), nullable=False))
+    model: str = Field(max_length=256, nullable=False)
```

Comment:
the question is, is the model name really enough to specify fully how the embedding was done? how about tokenizer settings? or text cleaning settings? or what happens if the text is too long? etc.

Reply:
Since all text-porcessing is done at the HF-endpoint it would IMO be sufficient to have a key that identifies the used HF feature-extraction pipeline. If something like `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v` is stored in `model` I think that would be sufficient.